config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  
+  get "/projects", to: "projects#index" 
```

Comment:
I think this route is not being used in your application and thus would be considered an unimplemented route. We do not want to add unimplemented routes in our applications because they open us open to cyber attacks.